### PR TITLE
Merge in 3.2.7

### PR DESCRIPTION
**Pull request by wilsonge**
Fixes https://github.com/beberlei/assert/issues/309 

I've done my best to resolve conflicts. But hopefully the unit tests will point out if I've missed anything along the way hopefully!

Also once this is merged can you guys update the changelog for version 3.3.0 then release 3.3.1 :)

### CODE REVIEW

**Comment by danieleartico** [comment]
Hello, any news about merging this PR, please? May I help somehow? :)

**Comment by wilsonge** [comment]
Ping to @oqq and @beberlei as this is kinda frustrating for several linked projects

**Comment by thirsch** [comment]
Any news here?

File: lib/Assert/LazyAssertion.php [hunk 1]
```diff
@@ -189,9 +195,15 @@ public function verifyNow(): bool
     }
 
     /**
+<<<<<<< HEAD
```
Comment (thirsch) [inline lib/Assert/LazyAssertion.php:198]: Looks like unresolved conflicts here and below.

Comment (wilsonge) [reply lib/Assert/LazyAssertion.php]: Ta fixed

**Comment by beberlei** [comment]
@wilsonge thank you!

**Pull request merged**